urlcache.go

```diff
@@ -80,20 +80,23 @@ func (c *urlcache) keepCurrent(currentDate time.Time) {
 		headResp, err := http.Head(c.url)
```

Comment:
@oxtoacart easylist is using it for now. It doesn't matter too much. We can change it when required.